src/Neo/ProtocolSettings.cs

```diff
@@ -58,19 +58,23 @@ public record ProtocolSettings
         public string[] SeedList { get; init; }
 
         /// <summary>
-        /// Indicates the time in milliseconds between two blocks.
+        /// The milliseconds between two block in NEO.
+        /// For code that needs the accurate block generation time based on blockchain state, 
+        /// use NeoSystemExtensions.GetBlockGenTime extension method instead.
         /// </summary>
         public uint MillisecondsPerBlock { get; init; }
 
         /// <summary>
-        /// Indicates the time between two blocks.
+        /// Indicates the time between two blocks based on the protocol settings.
+        /// This returns a fixed value from protocol settings. For dynamic block time based on blockchain state, 
+        /// use NeoSystemExtensions.GetBlockGenTime extension method instead.
         /// </summary>
         public TimeSpan TimePerBlock => TimeSpan.FromMilliseconds(MillisecondsPerBlock);
 
         /// <summary>
         /// The maximum increment of the <see cref="Transaction.ValidUntilBlock"/> field.
         /// </summary>
-        public uint MaxValidUntilBlockIncrement => 86400000 / MillisecondsPerBlock;
+        public uint MaxValidUntilBlockIncrement => 86400000 / MillisecondsPerBlock; //TODO keep the same??
```

Comment:
```suggestion
        public uint MaxValidUntilBlockIncrement => 86400000 / MillisecondsPerBlock;
```
It's OK, it's still timestamp for 24 hrs.